src/test/java/org/commcare/formplayer/tests/SubmitServiceTests.java

```diff
@@ -1,80 +0,0 @@
-package org.commcare.formplayer.tests;
-
-import org.commcare.formplayer.web.client.RestTemplateConfig;
-import org.commcare.formplayer.repo.FormSessionRepo;
-import org.commcare.formplayer.services.CategoryTimingHelper;
-import org.commcare.formplayer.services.RestoreFactory;
-import org.commcare.formplayer.services.SubmitService;
-import org.junit.jupiter.api.BeforeEach;
-import org.junit.jupiter.api.Test;
-import org.mockito.Mockito;
-import org.springframework.beans.factory.annotation.Autowired;
-import org.springframework.beans.factory.config.BeanFactoryPostProcessor;
-import org.springframework.boot.test.autoconfigure.web.client.RestClientTest;
-import org.springframework.boot.test.context.TestConfiguration;
-import org.springframework.boot.test.mock.mockito.MockBean;
-import org.springframework.context.annotation.Bean;
-import org.springframework.context.support.SimpleThreadScope;
-import org.springframework.http.HttpStatus;
-import org.springframework.test.web.client.MockRestServiceServer;
-import org.springframework.web.client.RestTemplate;
-import org.springframework.web.context.WebApplicationContext;
-
-import static org.assertj.core.api.Assertions.assertThat;
-import static org.mockito.Matchers.anyString;
-import static org.mockito.Mockito.when;
-import static org.springframework.test.web.client.match.MockRestRequestMatchers.requestTo;
-import static org.springframework.test.web.client.response.MockRestResponseCreators.withBadRequest;
-
-@RestClientTest(components = {SubmitService.class, RestTemplateConfig.class})
-public class SubmitServiceTests {
```

Comment:
This class was only testing that the RestTemplate didn't raise exceptions. Since it will now raise exceptions and those exceptions are now being handled it's of no use.